### DISCUSSION TITLE
魏老师，我用的是mac系统，安装的时候有一个错误出现

### DISCUSSION THREAD

**Discussion by limengmaths** [weihuayi/fealpy#49] [Installation]
在运行第六步的时候出现了错误：

(base) limengdeMacBook-Pro:~ limeng$ cd fealpy
(base) limengdeMacBook-Pro:fealpy limeng$ python setup_mac.py develop
running develop
running egg_info
error: [Errno 13] Permission denied
(base) limengdeMacBook-Pro:fealpy limeng$

**Comment by scaomath** [comment]
You can either install under `user` or use `chown` to change permission to that folder.

**Comment by limengmaths** [comment]
Last login: Tue Jul  7 01:30:15 on ttys000
虽然第六步骤的时候出现上面的错误，但是第七步骤git pull出现“Already up to date.”，具体内容如下：
The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.
(base) limengdeMacBook-Pro:~ limeng$ cd fealpy
(base) limengdeMacBook-Pro:fealpy limeng$ git pull
Already up to date.
(base) limengdeMacBook-Pro:fealpy limeng$

**Comment by weihuayi** [comment]
@limengmaths 安装 FEALPy 时， macOS 下要这样子安装 `sudo python setup_mac.py develop`， 网上文档我已经更新了

**Comment by weihuayi** [comment]
@limengmaths 另外， git pull 是为了拉最新更新， 如果远程没有更新， 和你本地一致， 那么就会显示 Already up to date. 这不是错误，

**Comment by limengmaths** [comment]
@weihuayi 👌谢谢魏老师
